srcpkgs/element-desktop/template

```diff
@@ -18,8 +18,8 @@ _ghpage="https://github.com/vector-im"
 _archive="archive/v${version}.tar.gz"
 distfiles="${_ghpage}/element-desktop/${_archive}>element-desktop.tar.gz
  ${_ghpage}/element-web/${_archive}>element-web.tar.gz"
```

Comment:
can you expand these variables?

Reply:
done, was inherited from the previous maintainer